hnn_core/cell.py

```diff
@@ -325,10 +325,12 @@ def _create_sections(self, p_secs, topology):
             self.sections[sec_name] = sec
 
             h.pt3dclear(sec=sec)
+            h.pt3dconst(0, sec=sec)  # be explicit, see documentation
```

Comment:
is this needed when there is `h.define_shape`? I feel one is enough ...

Reply:
Neither are _really_ needed (as you pointed out, the code worked before)

* `pt3dconst` is set to '0' by default; I thought I'd make it explicit here to remind ourselves.
* from the [doc of pt3dadd](https://nrn.readthedocs.io/en/latest/python/modelspec/programmatic/topology/geometry.html?highlight=define_shape#pt3dadd):

> Note: When L is changed, h.define_shape() should be executed to adjust the 3-d info so that branches appear connected.

I think it hasn't been a problem because none of our dendrites branch. Or do the last too basal ones...? I'm uncertain.

Reply:
can I just put it on record that I ~hate~ strongly dislike Neuron documentation :) Instead of raising sane errors, the onus is on the user to make sure things are executed in certain order with statements like "this may happen if you do this, but it may not be so bad ...".

Reply:
I'll counter-sign that one, and you can pick an even stronger word if you'd like ;)